docs/v2/integrations/advanced-custom-fields.md

```diff
@@ -77,8 +77,8 @@ This is where we’ll start in PHP.
 ```php
 $post = Timber::get_post();
 
-if (isset($post->hero_image) && strlen($post->hero_image)) {
-    $post->hero_image = Timber::get_image($post->hero_image);
+if (isset($post->meta('hero_image')) && strlen($post->meta('hero_image'))) {
+    $post->meta('hero_image') = Timber::get_image($post->meta('hero_image'));
```

Comment:
```suggestion
    $post->hero_image = Timber::get_image($post->meta('hero_image'));
```
This will have to stay the same since we want to make $this->hero_image available directly in  the timber context.

Reply:
True, I changed it too naively